pytorch_lightning/accelerators/accelerator.py

```diff
@@ -358,20 +353,23 @@ def setup_optimizers(self, trainer: 'pl.Trainer') -> None:
         self.lr_schedulers = lr_schedulers
         self.optimizer_frequencies = optimizer_frequencies
 
-    def setup_training_type_plugin(self, plugin: TrainingTypePlugin, model: 'pl.LightningModule') -> None:
+    def setup_training_type_plugin(self, model: 'pl.LightningModule') -> None:
         """Attaches the training type plugin to the accelerator."""
-        plugin.setup(model)
+        self.training_type_plugin.setup(model)
```

Comment:
do we need to change this in this PR?
same for precision below

Reply:
I can't remember why I made this change.. was getting some warning in my new IDE regarding passing the class attributes within class method, so updated this. If not required I will revert it, but still `plugin` argument is not required since it's basically `self.training_type_plugin`.